pkg/message_status_report/message_status_report.go

```diff
@@ -0,0 +1,98 @@
+package message_status_report
```

Comment:
Can we use the existing pacs.002 writer?

https://github.com/moov-io/rtp20022/blob/master/pkg/msg_status/write.go